resources/configurationFiles/analysisIndelCalling.xml

```diff
@@ -136,13 +129,11 @@
 
         <!--## Filter criteria -->
         <!--## The rows with corresponding values larger than below criteria will be filtered out. -->
-        <cvalue name="CRIT_ExAC_maxMAF" value="1.0" type="float"/>
-        <cvalue name="CRIT_EVS_maxMAF" value="1.0" type="float"/>
         <cvalue name="CRIT_GNOMAD_EXOMES_maxMAF" value="0.001" type="float"/>
         <cvalue name="CRIT_GNOMAD_GENOMES_maxMAF" value="0.001" type="float"/>
         <cvalue name="CRIT_1KGENOMES_maxMAF" value="0.01" type="float"/>
         <cvalue name="CRIT_RECURRENCE" value="7" type="integer"/>
-        <cvalue name="CRIT_LOCALCONTROL_maxMAF" value="0.01" type="float"/>
+        <cvalue name="CRIT_LOCALCONTROL_maxMAF" value="0.05" type="float"/>
```

Comment:
What is the effect of this change on the results of a workflow run? Does the workflow still produce identical results for identical inputs?

Reply:
This will relax the filtering in the no-control workflows. So will not be backward compatibility.

Reply:
This change should certainly be mentioned in the changelogs, then.

And it means, we can upgrade existing projects to this version of the workflow, as long as we fix this cutoff to the old value, right?

Reply:
BTW The whole PR is on the master! This means that we anyway would not apply this version to existing projects. The master is 2.1.0+ according to the tags.